packages/constants/src/constants.ts

```diff
@@ -4,9 +4,19 @@ export const NEXT_NAVIGATION_ID_ERROR = "";
 export const ANSWER_DISPLAY_VALUE_PLACEHOLDER = "_____";
 export const ANSWER_DISPLAY_VALUE_PLACEHOLDER_A11Y = "blank";
 
-export const STR_WEBSITE_NAME = "2024 BC Building Code";
+export const STR_WEBSITE_NAME = "2024 BC Building Code (Beta)";
```

Comment:
Do we want to start doing semantic versioning for our releases or is that premature